webapp/src/types/common/index.d.ts

```diff
@@ -1,7 +1,9 @@
 /**
  * Keep all common types here which are to be used throughout the project
 */
-type EventType = 'workitem.created' | 'workitem.updated' | 'workitem.deleted' | 'workitem.commented' | 'git.pullrequest.created'| 'git.pullrequest.updated' | 'ms.vss-code.git-pullrequest-comment-event' | 'git.push' | 'git.pullrequest.merged'
+type EventTypeBoards = 'workitem.created' | 'workitem.updated' | 'workitem.deleted' | 'workitem.commented'
+type EventTypeRepos = 'git.pullrequest.created'| 'git.pullrequest.updated' | 'ms.vss-code.git-pullrequest-comment-event' | 'git.push' | 'git.pullrequest.merged'
```

Comment:
You should use enums for these. You can add a TODO comment if you don't want to do it now.